backend/typescript/services/implementations/adminService.ts

```diff
@@ -108,6 +114,52 @@ class AdminService implements IAdminService {
     }
     return form;
   }
+
+  async addQuestionToTemplate(
+    formQuestion: CreateFormQuestionDTO,
+  ): Promise<FormQuestionDTO> {
+    const session = await mongoose.startSession();
+    session.startTransaction();
+
+    try {
+      // Get the current formTemplate and its list of questions
+      const formTemplate = await MgFormTemplate.findOne();
+      if (!formTemplate)
+        throw new Error("Failed to retrieve the form template");
+
+      const newFormQuestion = await MgFormQuestion.create({
+        type: formQuestion.type,
+        question: formQuestion.question,
+        required: formQuestion.required,
+        description: formQuestion.description,
+        options: formQuestion.options,
+        category: formQuestion.category,
+      });
+
+      formTemplate.formQuestions.push(newFormQuestion._id);
```

Comment:
I've completely forgotten, what's the difference between `_id` and `id`?

Reply:
the id is the string representation and _id is the actual OjectId type